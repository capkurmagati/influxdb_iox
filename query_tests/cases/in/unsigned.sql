-- Test basic SQL functionality of unsigned type
-- IOX_SETUP: TwoMeasurementsUnsignedType

-- count has type "unsigned" in line protocol
SELECT town, count from restaurant;

SELECT town, count from school
