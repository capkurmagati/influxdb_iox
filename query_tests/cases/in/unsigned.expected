-- Test Setup: TwoMeasurementsUnsignedType
-- SQL: SELECT town, count from restaurant;
+---------+-------+
| town    | count |
+---------+-------+
| andover | 40000 |
| reading | 632   |
+---------+-------+
-- SQL: SELECT town, count from school
+---------+-------+
| town    | count |
+---------+-------+
| andover | 25    |
| reading | 17    |
+---------+-------+
